### PR TITLE
`General`: Improve German language consistency

### PR DESCRIPTION
**Pull request by egekurt123**
#### Client
- [x] I **strictly** followed the [client coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client/).
- [x] Following the [theming guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client-design/), I specified colors only in the theming variable files and checked that the changes look consistent in both the light and the dark theme.

### Motivation and Context

There are Inconsistencies in the use of the formal "Sie" and the informal "du" across our platform's German translations. Preferring the use of "du" is essential to maintain a personal and less formal tone, aligning with our goal to make the platform more welcoming and accessible. Correcting these inconsistencies will enhance user experience and ensure clearer communication. 

#### Review
- [x] Review 1
- [x] Review 2
- [x] Review 3

## Summary by CodeRabbit

- **Documentation**
	- Updated the German language guidelines to recommend using "Du" for a more personal and informal tone.

### CODE REVIEW

File: src/main/webapp/i18n/de/featureOverview.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
...
+                    "descriptionTextOne": "Ohne eine Aufsicht, kann es sich so anfühlen, als könntest du deine Studierenden nicht genau beobachten. Um dem entgegenzuwirken, beinhaltet Artemis eine Überwachung der Klausuren. Für alle Studierenden, die an der Klausur teilnehmen, generiert Artemis eine Klausursitzung. Die Klausursitzung enthält neben anderen Feldern, wann und wo die Klausur gestartet wurde, die öffentliche IP-Adresse und einen einzigartigen Fingerabdruck, der auf dem Browser und dem Betriebssystem basiert. Zusätzlich generiert die Klausur eine eindeutige Kennung, die mit jeder Anfrage gesendet wird. Studierende haben die Möglichkeit, während der Klausur zwischen den Computern zu wechseln und auch offline weiterzuarbeiten. Als Ausbilder hast Du die Möglichkeit, alle Sitzungen und das Verhalten der Studierenden zu analysieren und verdächtig erscheinende Fälle zu markieren."
```
Comment (Strohgelaender) [inline src/main/webapp/i18n/de/featureOverview.json:42]: imo this sounds a bit strange:
```suggestion
                    "descriptionTextOne": "Ohne eine Aufsicht kann es sich so anfühlen, als würdest du deine Studierenden nicht genau beobachten. Um dem entgegenzuwirken, beinhaltet Artemis eine Überwachung der Klausuren. Für alle Teilnehmenden generiert Artemis eine Klausursitzung. Diese Klausursitzung enthält unter anderem, wann und wo die Klausur gestartet wurde, die öffentliche IP-Adresse und einen einzigartigen Fingerabdruck, der auf dem Browser und dem Betriebssystem basiert. Zusätzlich generiert die Klausur eine eindeutige Kennung, die mit jeder Anfrage gesendet wird. Studierende haben die Möglichkeit, während der Klausur zwischen den Computern zu wechseln und auch offline weiterzuarbeiten. Als Ausbilder hast Du die Möglichkeit, alle Sitzungen und das Verhalten der Studierenden zu analysieren und verdächtig erscheinende Fälle zu markieren."
```

File: src/main/webapp/i18n/de/lti.json [hunk 2]
```diff
@@ -89,14 +89,14 @@
                 "link": "Verlinken",
                 "unknownError": "Aufgrund eines unbekannten Fehlers nicht erfolgreich. Bitte kontaktiere einen Admin!"
             },
-            "missingConfigurationWarning": "Fehlende Werte in der LTI1.3-Konfiguration. Starts werden nicht funktionieren. Bitte wenden Sie sich für die Konfiguration an den Administrator.",
+            "missingConfigurationWarning": "Fehlende Werte in der LTI1.3-Konfiguration. Starts werden nicht funktionieren. Bitte wende dich für die Konfiguration an den Administrator.",
             "selectContentFromCourse": "Wähle Inhalte aus dem Kurs {{ title }} aus",
             "deletePlatformError": "Die LTI-Plattform konnte nicht gelöscht werden",
-            "selectContentTooltip": "Wählen Sie einfach Ihre bevorzugten Übungen aus und klicken Sie dann auf die Schaltfläche Link, um sie in die Plattform zu integrieren.",
+            "selectContentTooltip": "Wähle einfach Ihre bevorzugten Übungen aus und klicke dann auf die Schaltfläche Link, um sie in die Plattform zu integrieren.",
```
Comment (Strohgelaender) [inline src/main/webapp/i18n/de/lti.json:95]:
```suggestion
            "selectContentTooltip": "Wähle einfach deine bevorzugten Übungen aus und klicke dann auf die Schaltfläche Link, um diese in die Plattform zu integrieren.",
```

File: src/main/webapp/i18n/de/programmingExercise.json [hunk 3]
```diff
@@ -444,7 +444,7 @@
             "reEvaluateSuccessful": "{{number}} automatischen Ergebnisse wurden erfolgreich aktualisiert, einschließlich der Vorlage und der Lösung.",
             "reEvaluateFailed": "Die Neubewertung ist fehlgeschlagen: {{message}}",
             "editSelectedModal": {
-                "currentlyEdited": "Sie bearbeiten die folgenden Übungen:",
+                "currentlyEdited": "D bearbeitest die folgenden Übungen:",
```
Comment (Strohgelaender) [inline src/main/webapp/i18n/de/programmingExercise.json:447]:
```suggestion
                "currentlyEdited": "Du bearbeitest die folgenden Übungen:",
```

File: docs/dev/guidelines/language-guidelines.rst [hunk 4]
```diff
@@ -52,6 +52,7 @@ The German language is mainly used for user interaction.
 **Best practices** (written in German for demonstration purposes)
 
 * **Nutze neutrale Formulierungen**: Wenn möglich ist es empfehlenswert, neutrale Formulierungen zu finden. Neben passiven Formen, wie z.B. Studierende, können auch aktive mit „Mensch“ bzw. „Person“ etc. verwendet werden. Das spart „der/die“ und inkludiert alle Menschen unabhängig davon, welche Eigenschaften/Kategorien diese Personen sonst noch mitbringen.
+* **Verwende die Anrede „Du“**:Um eine persönliche und weniger formelle Atmosphäre zu schaffen, verwende „Du“ statt „Sie".
```
Comment (b-fein) [inline docs/dev/guidelines/language-guidelines.rst:55]:
```suggestion
* **Verwende die Anrede „du“**:Um eine persönliche und weniger formelle Atmosphäre zu schaffen, verwende „du“ statt „Sie".
```
Existing translations use lowercase ‘du’, ‘dich’, …

Please adapt the updated translations in this PR to follow this schema as well.

**Review by b-fein — CHANGES_REQUESTED** [review]

File: src/main/webapp/i18n/de/aboutUs.json [hunk 5]
```diff
@@ -51,7 +51,7 @@
                 "text": "<a href={{ learningAnalyticsUrl }}>Lernanalytik:</a> Statistiken, mit denen sich die Studierenden mit dem Kursdurchschnitt vergleichen können. Bewerten der durchschnittliche Leistung der Studierenden anhand von Aufgaben und Kompetenzen."
             },
             "adaptiveLearning": {
-                "text": "<a href={{ adaptiveLearningUrl }}>Adaptives Lernen:</a> Definieren Sie Kompetenzen und Lernziele. Verfolgen Sie den Fortschritt der Studierenden und geben Sie individuelles Feedback."
+                "text": "<a href={{ adaptiveLearningUrl }}>Adaptives Lernen:</a> Definiere Kompetenzen und Lernziele. Verfolge den Fortschritt der Studierenden und gebe individuelles Feedback."
```
Comment (rabeatwork) [inline src/main/webapp/i18n/de/aboutUs.json:54]:
```suggestion
                "text": "<a href={{ adaptiveLearningUrl }}>Adaptives Lernen:</a> Definiere Kompetenzen und Lernziele. Verfolge den Fortschritt der Studierenden und gib individuelles Feedback."
```

File: src/main/webapp/i18n/de/lti.json [hunk 6]
```diff
@@ -47,22 +47,22 @@
...
                 "clientIdTooltip": "Es ist möglich, dieses Feld und die folgenden URI-Felder manuell zu konfigurieren, aber es wird dringend empfohlen, den Dynamic Registration Service zur automatischen Konfiguration zu verwenden.",
                 "form": {
                     "userPrefix": {
                         "forbidden": "Nutzer-Präfix darf keine Sonderzeichen beinhalten."
                     }
                 },
-                "customNameIdTooltip": "Geben Sie einen erkennbaren Namen für die Plattform ein, um sie in der Liste leicht identifizieren zu können. Dieser Name dient nur als administrative Referenz und hat keinen Einfluss auf die tatsächliche Konfiguration oder Konnektivität der Plattform."
+                "customNameIdTooltip": "Gebe einen erkennbaren Namen für die Plattform ein, um sie in der Liste leicht identifizieren zu können. Dieser Name dient nur als administrative Referenz und hat keinen Einfluss auf die tatsächliche Konfiguration oder Konnektivität der Plattform."
```
Comment (rabeatwork) [inline src/main/webapp/i18n/de/lti.json:65]:
```suggestion
                "customNameIdTooltip": "Gib einen erkennbaren Namen für die Plattform ein, um sie in der Liste leicht identifizieren zu können. Dieser Name dient nur als administrative Referenz und hat keinen Einfluss auf die tatsächliche Konfiguration oder Konnektivität der Plattform."
```

File: src/main/webapp/i18n/de/lti.json [hunk 7]
```diff
@@ -89,14 +89,14 @@
                 "link": "Verlinken",
                 "unknownError": "Aufgrund eines unbekannten Fehlers nicht erfolgreich. Bitte kontaktiere einen Admin!"
             },
-            "missingConfigurationWarning": "Fehlende Werte in der LTI1.3-Konfiguration. Starts werden nicht funktionieren. Bitte wenden Sie sich für die Konfiguration an den Administrator.",
+            "missingConfigurationWarning": "Fehlende Werte in der LTI1.3-Konfiguration. Starts werden nicht funktionieren. Bitte wende dich für die Konfiguration an den Administrator.",
             "selectContentFromCourse": "Wähle Inhalte aus dem Kurs {{ title }} aus",
             "deletePlatformError": "Die LTI-Plattform konnte nicht gelöscht werden",
-            "selectContentTooltip": "Wählen Sie einfach Ihre bevorzugten Übungen aus und klicken Sie dann auf die Schaltfläche Link, um sie in die Plattform zu integrieren.",
+            "selectContentTooltip": "Wähle einfach deine bevorzugten Übungen aus und klicke dann auf die Schaltfläche Link, um diese in die Plattform zu integrieren.",
```
Comment (rabeatwork) [inline src/main/webapp/i18n/de/lti.json:95]:
```suggestion
            "selectContentTooltip": "Wähle deine bevorzugten Übungen aus und klicke dann auf die Schaltfläche Link, um diese in die Plattform zu integrieren.",
```

File: src/main/webapp/i18n/de/quizPool.json [hunk 8]
```diff
@@ -6,9 +6,9 @@
             "groupName": "Gruppenname",
             "addGroup": "Gruppe hinzufügen",
             "delete": {
-                "question": "Möchten Sie die Quizgruppe <strong>{{ title }}</strong> wirklich löschen? Alle zugewiesenen Fragen werden nicht zugeordnet."
+                "question": "Möchtest du die Quizgruppe <strong>{{ title }}</strong> wirklich löschen? Alle zugewiesenen Fragen werden nicht zugeordnet."
             },
-            "dragExplanation": "Ziehen Sie die Frage unten, um sie einer Gruppe zuzuordnen.",
+            "dragExplanation": "Ziehe die Frage unten, um sie einer Gruppe zuzuordnen.",
```
Comment (rabeatwork) [inline src/main/webapp/i18n/de/quizPool.json:11]:
```suggestion
            "dragExplanation": "Ziehe die unten stehende Frage, um sie einer Gruppe zuzuordnen.",
```

**Review by edkaya — DISMISSED** [review]
Looks good to me

**Review by b-fein — DISMISSED** [review]

**Review by Hialus — CHANGES_REQUESTED** [review]
I've went through the language files and found some missed instances:
- File `importUser.json`: `artemisApp.importUsers.usersForImport.importFailed` & `artemisApp.importUsers.usersForImport.failedRows`
- File `instructorDashboard.json`: `...exportRepos.questions...` (both) & `...exportSubmissions.question`
- I found some other things when searching for `ihr` that may also be relevant, but I did not have the time to go through all the results.

File: src/main/webapp/i18n/de/importUsers.json [hunk 9]
```diff
@@ -8,12 +8,12 @@
...
+                "tooltip": "Bitte wähle die .csv Datei mit den Nutzern zum Import aus. Pro Eintrag muss entweder die 'LOGIN', 'REGISTRATION_NUMBER' oder 'E-MAIL' Spalte ausgefüllt sein. Folgendes Format muss dabei beachtet werden: Login, Matrikelnummer, E-Mail, Vorname, Nachname. Du musst diese in der obersten Spalte angeben!"
             },
             "usersForImport": {
                 "label": "Nutzer zum Importieren",
                 "tooltip": "Diese Nutzer werden importiert beim Klicken auf \"Importieren\".",
-                "importFailed": "Bitte versuchen sie es nach dem reparieren der Datei erneut, die Datei konnte wegen folgendem Fehler nicht gelesen werden:",
+                "importFailed": "Bitte versuche es nach dem reparieren der Datei erneut, die Datei konnte wegen folgendem Fehler nicht gelesen werden:",
                 "failedRows": "Die folgenden Reihen in ihrer CSV sind ungültig und müssen entweder den Login, eine Matrikelnummer oder eine E-Mail enthalten:",
```
Comment (Hialus) [inline src/main/webapp/i18n/de/importUsers.json:17]: You still missed this. It should be "Die folgenden Reihen in deiner CSV sind ungültig"

**Review by Hialus — CHANGES_REQUESTED** [review]
I've gone through the "ihr" search now and found 2 instances where "ihr" should be replaced with some for of "dein"
- `programmingExercise.json`: `/artemisApp/programmingExercise/reset/question` should say `Um deine Auswahl...`

**Review by rabeatwork — APPROVED** [review]

**Review by b-fein — APPROVED** [review]

**Review by edkaya — APPROVED** [review]
Reapprove

**Pull request merged**